README.md

```diff
@@ -22,11 +22,20 @@ nrfjprog -f nrf52 --program nrf802154_sniffer/nrf802154_sniffer.hex --chiperase
 ```
 
 ### Install extcap script
+
+#### Linux
 Copy the provided script to the appropriate directory:
 ```
 sudo cp nrf802154_sniffer/nrf802154_sniffer.py /usr/lib/x86_64-linux-gnu/wireshark/extcap/
 ```
 
+#### Windows
+Copy both .bat and .py scripts from nrf802154_sniffer to the appropriate directory:
```

Comment:
not: let say nrf802154_sniffer.bat and nrf802154_sniffer.py instead of pointing to extension itself.